contentcuration/contentcuration/frontend/shared/data/serverSync.js

```diff
@@ -379,3 +376,8 @@ export function stopSyncing() {
   // Dexie's slightly counterintuitive method for unsubscribing from events
   db.on('changes').unsubscribe(handleChanges);
 }
+
+export function forceServerSync() {
+  debouncedSyncChanges();
+  debouncedSyncChanges.flush();
```

Comment:
We should make this return a promise here, so that we can chain other actions from it elsewhere. Just returning the final line should be sufficient.